tests/pass-dep/libc/libc-pipe.rs

```diff
@@ -110,3 +112,11 @@ fn test_pipe_array() {
     let mut fds: [i32; 2] = [0; 2];
     assert_eq!(unsafe { pipe(&mut fds) }, 0);
 }
+
+// Test if O_NONBLOCK flag is supported.
```

Comment:
```suggestion
// Test if pipe2 (including the O_NONBLOCK flag) is supported.
```

Reply:
Is it necessary to use doc comment in the test? But sure I will commit it first.

Reply:
Not *necessary* but it's generally a good idea to use a doc comment when documenting an item. It expresses intent. And since I was changing the comment anyway I made it a doc comment while I am at it.

Reply:
Alright, thanks for the explanation!